test/chat-plugins/trivia.js

```diff
@@ -27,32 +14,31 @@ let NumberModeTrivia;
 
 function makeUser(name, connection) {
 	let user = new User(connection);
-	user.name = name;
-	user.userid = name.toLowerCase().replace(/[^a-z0-9-]+/g, '');
+	user.forceRename(name, true);
+	user.connected = true;
 	Users.users.set(user.userid, user);
 	user.joinRoom('trivia', connection);
 	return user;
 }
 
 function destroyUser(user) {
-	if (user.connected) {
-		user.disconnectAll();
-		user.destroy();
-	}
+	if (!user || !user.connected) return false;
+	user.resetName();
+	user.disconnectAll();
+	user.destroy();
 }
 
 describe('Trivia', function () {
 	before(function () {
-		// ...until we can load the trivia module right before the tests begin,
-		// where the context contains the globals missing from when this was
-		// first defined.
-		triviaModule.load(triviaModule.id);
-
-		SCORE_CAPS = triviaModule.exports.SCORE_CAPS;
-		Trivia = triviaModule.exports.Trivia;
-		FirstModeTrivia = triviaModule.exports.FirstModeTrivia;
-		TimerModeTrivia = triviaModule.exports.TimerModeTrivia;
-		NumberModeTrivia = triviaModule.exports.NumberModeTrivia;
+		// The trivia module cannot be loaded outside of the scope of the unit tests,
+		// since global variables in the outermost scope of the module are undefined in
+		// this scope.
```

Comment:
`Config` being referred to outside of any functions or anything breaks it when I try to require the module outside of the `describe` block in the test, since it's undefined at that point